lexis/render/clipPlanes.fbs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+
+// This event is used to communicate clip planes between
+// applications. Every plane is defined in point normal
+// form. n normal and d value [ nx, ny, nz d ]. Here for
+// given points, p, n.p + d = 0 is satisfied.
```

Comment:
You have to indicate which hemispace is the clipped one, that the side pointed by the normal or the opposite one?